src/bootstrap/test.rs

```diff
@@ -834,22 +834,68 @@ impl Step for RustdocJSNotStd {
     }
 }
 
-fn check_if_browser_ui_test_is_installed_global(npm: &Path, global: bool) -> bool {
+fn get_browser_ui_test_version_from_string(input: &String) -> Option<String> {
```

Comment:
Same, this is out of scope for this PR. Please open another one for checking the package version.

Reply:
Sure, will do :+1: